src/components/display/DocxDisplay.tsx

```diff
@@ -12,17 +11,22 @@ const MAMMOTH_OPTIONS = {
   ),
 };
 
-const styles = {
+const styles: Record<string, CSSProperties> = {
   container: {
     maxWidth: 960, // Maximum width to roughly a nice page
     overflowX: "auto",
   },
 };
 
-const DocxDisplay = ({ contents, loading }) => {
+type DocxDisplayProps = {
+  contents: Blob;
```

Comment:
`contents?: Blob;` instead?
We could also have a reusable generic props type for file content display, since we are using the same props in the other components.